tests/flags/pass_lee.js

```diff
@@ -1,13 +1,13 @@
-const flags = ({ fl = { h: 'help' }, description = {}, ...input }) => {
-  !input.hasOwnProperty('help') && (description = { ...input })
+const formDes = (flag, input) => `-${flag[0]}, --${flag}: ${input[flag]}`
 
-  for (let [key, value] of Object.entries(input)) {
-    if (key === 'help') {
-      value.forEach((ele) => {
-        description[ele] = input[ele]
-      })
-    }
-    fl[key[0]] = key
+const flags = ({ descriptions, ...input }) => {
```

Comment:
Why do you destructure descriptions here ?
destructuring is not to be use to create variables but "extract" values form the object.

You never care about the property `descriptions` of your inputs.
You should write:

```js
const flags = (input) => {
  const descriptions = input.hasOwnProperty('help')
    ? input.help.map((flag) => formDes(flag, input))
    : Object.keys(input).map((flag) => formDes(flag, input))
```

Reply:
here destructuring is a prefect fit for the `help` property

Reply:
ok ok thanks